lib/mshoplib/src/MShop/Service/Provider/Payment/PayPalExpress.php

```diff
@@ -727,8 +731,8 @@ protected function getOrderDetails( \Aimeos\MShop\Order\Item\Base\Iface $orderBa
 				foreach( $orderBase->getService( 'delivery' ) as $service )
 				{
 					$deliveryPrices = $this->addPrice( $deliveryPrices, $service->getPrice() );
-
-					$values['L_SHIPPINGOPTIONAMOUNT' . $lastPos] = number_format( $service->getPrice()->getCosts(), 2, '.', '' );
+					
```

Comment:
There are several trailing tab. Can you please remove them?

Reply:
Sorry, fixed this.